scripts/create-app.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config();
+
+const api = require('./helpers/ts-api');
+const chalk = require('chalk');
+const templates = require('./helpers/templates');
+const utils = require('./helpers/utils');
+
+const TUNNEL_DOMAIN = 'localtunnel.me';
+
+const logSuccess = ({ appName, appEmbedUrl, appStoreUrl }) => {
```

Comment:
Don't think I've seen this syntax before, what's putting the curly braces around the arguments do in this context? Is it expecting a single object and dereferencing these three object properties so they can be used as normal variables in our function?